src/tests/lib/repositories/impl_no_sql/order_status_history_impl_test.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+from unittest import mock
+
+from mongoengine import connect, disconnect
+
+from src.constants.audit import Status
+from src.constants.order_status import OrderStatus
+from src.lib.entities.mongo_engine_odm_mapping import OrderStatusHistory
+from src.lib.repositories.impl_no_sql.order_status_history_repository_impl import (
+    OrderStatusHistoryRepositoryImpl,
+)
+from src.tests.utils.fixtures.mapping_odm_fixtures import (
+    build_order_status_histories,
+    build_order_status_history,
+)
+
+
+class OrderStatusHistoryRepositoryImplTest(unittest.TestCase):
```

Comment:
Maybe you could potentially create a mongo base class for testing.

Not pretty sure but feels like you need to do it for future tests

Reply:
done.